test_agent/python/testagent.py

```diff
@@ -65,21 +61,22 @@
 logger = logging.getLogger("File:Line# Debugger")
 logger.setLevel(logging.DEBUG)
 
+RESPONSE_URI = UUri(ue_id=1, ue_version_major=1, resource_id=0)
+
 
 class SocketUListener(UListener):
     def on_receive(self, umsg: UMessage) -> None:
         logger.info("Listener received")
         if umsg.attributes.type == UMessageType.UMESSAGE_TYPE_REQUEST:
-            attributes = UAttributesBuilder.response(
+            message = UMessageBuilder.response(
                 umsg.attributes.sink,
                 umsg.attributes.source,
-                UPriority.UPRIORITY_CS4,
                 umsg.attributes.id,
             ).build()
             any_obj = any_pb2.Any()
             any_obj.Pack(StringValue(value="SuccessRPCResponse"))
             res_msg = UMessage(
-                attributes=attributes,
+                attributes=message,
```

Comment:
shouldn't this then be attributes=message.attributes ?

Reply:
yeah you're probably right... but I do fix this in a future PR

Reply:
yeah made it work but this code is unused in this PR, fixed in subsequent PR